homeassistant/components/habitica/strings.json

```diff
@@ -15,26 +15,39 @@
     },
     "step": {
       "user": {
+        "title": "Habitica - Gamify your life",
         "menu_options": {
           "login": "Login to Habitica",
           "advanced": "Login to other instances"
         },
-        "description": "Connect your Habitica profile to allow monitoring of your user's profile and tasks."
+        "description": "![Habiticans](https://habitica.com/static/img/home-main@3x.ffc32b12.png) Connect your Habitica account to keep track of your adventurer's stats, progress, and manage your to-dos and daily tasks.\n\n[Don't have a Habitica account? Sign up here.](https://habitica.com/register)"
```

Comment:
We should not add URLs directly into our translations. The problem is that if these change, all translations are problematic and invalid.

Instead, URLs need to be injected as placeholders.

../Frenck

Reply:
@frenck done 👍